src/main/java/com/nyxei/pollingus/domain/messages/Name.java

```diff
@@ -0,0 +1,161 @@
+package com.nyxei.pollingus.domain.messages;
```

Comment:
Entity or message ?

Reply:
it's an entity, but I'm not using it so I'm going to delete it